common/uninstall.sh

```diff
@@ -18,7 +18,7 @@ fi
 ui_print " "
 ui_print "   Disabling Google's Flip to Shhh..."
 # Disabling Google's Flip to Shhh
-WELLBEING_PREF_FILE=$(find /data/data/com.google.android.apps.wellbeing*/shared_prefs -name "PhenotypePrefs.xml")
+WELLBEING_PREF_FILE=$(find /data/data/com.google.android.apps.wellbeing* -name "PhenotypePrefs.xml")
```

Comment:
Thought we were removing this

Reply:
removing what? flip to shhh? i didnt know

Reply:
Same as AutoDND